src/pybind/mgr/CMakeLists.txt

```
@@ -1,3 +1,10 @@
+set(mgr_module_install_excludes
+  REGEX "CMakeLists.txt" EXCLUDE
+  REGEX "\\.gitignore" EXCLUDE
+  REGEX "tox.ini" EXCLUDE
+  REGEX "requirements.*\.txt" EXCLUDE
+  REGEX "constraints.*\.txt" EXCLUDE)
```

For file-like matching I think `PATTERN` is easier and safer to use (in fact some of this regex don't escape `.`):
```suggestion
  PATTERN "CMakeLists.txt" EXCLUDE
  PATTERN ".gitignore" EXCLUDE
  PATTERN "tox.ini" EXCLUDE
  PATTERN "requirements*.txt" EXCLUDE
  PATTERN "constraints*.txt" EXCLUDE)
```

---

updated.

---

Thanks!